cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -4,6 +4,10 @@
 default["bcpc"]["hadoop"]["zabbix"]["mail_source"] = "zabbix.zbx_mail.sh.erb"
 default["bcpc"]["hadoop"]["zabbix"]["cookbook"] = nil 
 
+# Interval within which chef-client is expected to run
+default["bcpc"]["hadoop"]["zabbix"]["chef_client_check_interval"] =
+  "#{((node['chef_client']['interval'].to_i + node['chef_client']['splay'].to_i) * 2 / 60).ceil}m"
```

Comment:
Probably overboard for now:
In the future, I could see different machines with different run-times (e.g. a worker may run quickly but maintenance tasks may blow up run time every once and a while). As such it may make sense for each Chef run to talk to Graphite to get its last execution time or some simple statistics to compute how long it _should_ take (we could even dynamically kick the trigger out before running a long-running task). Otherwise, we could query our node object for the last `:ohai_time` on the node object for when it last started a Chef run (could keep an array for summary statistics).

For now just a Chef attribute should be awesome though.